apps/opentelemetry_experimental/src/opentelemetry_experimental.app.src

```diff
@@ -1,6 +1,6 @@
 {application, opentelemetry_experimental,
  [{description, "Implementation of unstable OpenTelemetry signals"},
-  {vsn, "0.5.4-emqx"},
+  {vsn, "0.5.5-emqx"},
```

Comment:
we should try to avoid non numbers in the vsn string.
maybe use a fourth number if there is still hop syncing it to upstream, otherwise diverge from upstream.